src/pages/docs/ai-transport/index.mdx

```diff
@@ -4,3 +4,56 @@ meta_description: "Learn more about Ably's AI Transport and the features that en
 redirect_from:
   - /docs/products/ai-transport
 ---
+
+Ably AI Transport is a solution for building stateful, steerable, multi-device AI experiences into new or existing applications. You can use AI Transport as the transport layer with any LLM or agent framework, without rebuilding your existing stack or being locked to a particular vendor.
+
+## Key features
+
+AI Transport builds on [Ably Pub/Sub](/docs/basics) to enable reliable interactive AI experiences at scale and provides the following key features:
+
+// TODO: Check this lines up with planned doc sections
+* [Reliable, resumable token streams](#reliable-tokens)
+* [Session management](#session)
+* [Complex message patterns](#message)
+* [Enterprise controls](#enterprise)
+
+### Reliable, resumable token streams <a id="reliable-tokens"/>
+
+Ably AI Transport enables you to [stream tokens](/docs/ai-transport/features/token-streaming) reliably from your AI agent to your client devices using a channel. The token stream survives reconnections and token history is available for any new clients that connect.
+
+### Session management <a id="session"/>
+
+With AI Transport, communication between the client and agent is not tied to the connection state of either party. This gives you much more control over how to manage session lifetime and opens up opportunities for improved user experience, for example push notifications to offline users when a long-running task completes.
+
+// TODO: links
+
+### Complex message patterns <a id="message"/>
+
+Truly interactive AI experiences require more than a simple HTTP request-response exchange between a single client and agent. AI transport allows the use of [complex messaging patterns](//TODO: Link here), for example:
```

Comment:
I'm guessing if this is meant to be where Advanced messaging is, the link would be `/docs/ai-transport/features/advanced-messaging` Yet to be created though.